kernel/kernel-component/src/main/webapp/WEB-INF/user-components.xml

```diff
@@ -116,4 +116,22 @@
 		<property name="memoryService" ref="org.sakaiproject.memory.api.MemoryService" />
 	</bean>
 
+	<bean id="org.sakaiproject.user.api.CandidateDetailProvider"
```

Comment:
This is relying on the load order of the component manager which is unspecified, although it does work at the moment I'm not sure it's a good design. If the kernel is ever loaded after the providers the wrong provider will get loaded.